.github/workflows/nightly_release.yaml

```diff
@@ -0,0 +1,33 @@
+name: nightly release
+
+on:
+  workflow_call:
+
+jobs:
+    releaseDate:   
```

Comment:
probably worth just making this a step under `createRelease`. It takes more time to do it the current way because it has to provision an entirely new runner.

Reply:
I think we cannot call a reusable workflow from a job step. `createRelese` and `switchBranch` both are using resuable workflows. That's why I've put `releaseDate` seperately.